src/OpenLoco/Windows/CompanyList.cpp

```diff
@@ -575,11 +573,7 @@ namespace OpenLoco::Ui::Windows::CompanyList
 
         if (window != nullptr)
         {
-            if (Input::isToolActive(_toolWindowType, _toolWindowNumber))
-            {
-                Input::toolCancel();
-                window = WindowManager::bringToFront(WindowType::companyList);
-            }
+            Input::toolCancel();
```

Comment:
I think you still need the bringToFront just incase the toolCancel abort event brings a different window to front.